applications/asset_tracker_v2/src/cloud/cloud_wrapper.h

```diff
@@ -210,6 +214,14 @@ int cloud_wrap_ui_send(char *buf, size_t len, bool ack, uint32_t id,
  */
 int cloud_wrap_cloud_location_send(char *buf, size_t len, bool ack, uint32_t id);
 
+/**
+ * @brief Indicates whether cloud implementation can and is willing to return the resolved
```

Comment:
Dunno whether this is a matter of willingness.
I'd rather leave it out like I suggested in the doc part.

Reply:
Well, it's not purely about the capability. Even if the cloud would be capable of providing the location back to the device in its APIs, it may well not do so due to other factors such as configured location method priority order, etc.
Whether "willigness" is a correct term or not is another question but it's not just about capability.

Reply:
How about something like "... can and is configured to return..."?

Reply:
I'll use it.

Reply:
Used also the same phrase in location_module.rst.

Reply:
I still see the word willing...?

Reply:
For some unknown reason, my `git push` didn't work properly although locally everything showed that they are aligned with the remote branch. After some tricks, it should be fine now.